### PR TITLE
Llamaindex: add tokenizer_id and support chat

### PR DESCRIPTION
**Pull request by ivy-lv11**
Llamaindex: add tokenizer_id and support chat

### CODE REVIEW

File: python/llm/example/CPU/LlamaIndex/README.md [hunk 1]
```diff
@@ -21,6 +17,10 @@ The RAG example ([rag.py](./rag.py)) is adapted from the [Official llama index R
     pip install llama-index-readers-file llama-index-vector-stores-postgres llama-index-embeddings-huggingface
     ```
 
+* **Install BigDL**
```
Comment (shane-huang) [inline python/llm/example/CPU/LlamaIndex/README.md:20]: Change BigDL to IPEX-LLM

File: python/llm/src/ipex_llm/llamaindex/llms/bigdlllm.py [hunk 2]
```diff
@@ -171,6 +173,10 @@ class BigdlLLM(CustomLLM):
             "`messages_to_prompt` that does so."
         ),
     )
+    load_low_bit: bool = Field(
```
Comment (shane-huang) [inline python/llm/src/ipex_llm/llamaindex/llms/bigdlllm.py:176]: Use an API style similar to langchain
IpexLLM.from_model_id()
IpexLLM.from_low_bit()

File: python/llm/example/CPU/LlamaIndex/README.md (hunk 1 above)
Comment (ivy-lv11) [reply python/llm/example/CPU/LlamaIndex/README.md]: Fixed.

File: python/llm/src/ipex_llm/llamaindex/llms/bigdlllm.py (hunk 2 above)
Comment (ivy-lv11) [reply python/llm/src/ipex_llm/llamaindex/llms/bigdlllm.py:176]: Fixed.

**Review by shane-huang — APPROVED** [review]
LGTM

**Pull request merged**